Makefile

```diff
@@ -82,7 +82,7 @@ $(GENERATE_UNBOUND_METHODS_EXAMPLE_SRCS): $(GENERATE_UNBOUND_METHODS_EXAMPLE_SPE
 		$(EXAMPLE_CLIENT_DIR)/generateunboundmethods/git_push.sh
 
 install:
-	go install github.com/bufbuild/buf/cmd/buf@v1.3.1
+	go install github.com/bufbuild/buf/cmd/buf@v1.12.0
```

Comment:
New `plugin:` key in buf.gen.yaml requires Buf CLI 1.8+: https://docs.buf.build/bsr/remote-plugins/migrating-from-alpha#bufgenyaml-changes.